webapp/src/lib/LeafletMapCoordinates.js

```diff
@@ -9,17 +9,17 @@ export default class LeafletMapCoordinates {
   toLatLngBounds(bounds) {
     let [lower, upper] = bounds
 
-    lower = this.cartesianToLatLng({ x: lower[0], y: lower[1] })
-    upper = this.cartesianToLatLng({ x: upper[0], y: upper[1] })
+    let a = this.cartesianToLatLng({ x: lower[0], y: lower[1] })
+    let b = this.cartesianToLatLng({ x: upper[0], y: upper[1] })
 
-    return new L.LatLngBounds(lower, upper)
+    return new L.LatLngBounds(a, b)
```

Comment:
While I do think that defining new variables here is a good idea (re-assigning arguments could be confusing) I'm sure we can find a better name for these two!